dsgrid/config/dimensions.py

```diff
@@ -694,3 +682,35 @@ def create_project_dimension_model(model, is_base):
     data = create_dimension_common_model(model).dict()
     data["is_base"] = is_base
     return ProjectDimensionModel(**data)
+
+
+def check_display_name(display_name: str):
+    """Check that a dimension display name meets all requirements.
+
+    Raises
+    ------
+    ValueError
+        Raised if the display_name is invalid.
+    """
+    if display_name == "":
+        raise ValueError("display_name cannot be empty")
+    if REGEX_VALID_REGISTRY_DISPLAY_NAME.search(display_name) is None:
+        raise ValueError(f"{display_name=} does not meet the requirements")
```

Comment:
Is this a new syntax for `f"display_name={display_name}"`?

Reply:
Yes. It is new in a recent python version.